client/my-sites/checklist/wpcom-checklist/component.jsx

```diff
@@ -432,6 +434,41 @@ class WpcomChecklistComponent extends PureComponent {
 		);
 	};
 
+	renderDomainVerifiedTask = ( TaskComponent, baseProps, task ) => {
+		const { translate, siteSlug } = this.props;
+
+		return (
+			<TaskComponent
+				{ ...baseProps }
+				bannerImageSrc="/calypso/images/stats/tasks/create-tagline.svg"
```

Comment:
This is just a tiny thing because I don't think this banner image ends up being shown anywhere, but should this either have its own `svg`, or otherwise be blank?